docs/ci-integration/remote-buildkit.md

```diff
@@ -0,0 +1,63 @@
+# Remote Buildkit
+
+## Introduction
+
+In some cases, you may want to run a remote instance of `earthly/buildkitd`. This guide is intended to help you identify if you might benefit from this configuration, and to help you set it up correctly.
+
+### Why Remote?
+
+Running a remote daemon is a unique feature of Earthly. It allows the build to happen elsewhere; even when executing it from your local development machine. However, it is not always the best option. Before setting up a remote daemon, first look into Earthly's shared caching capabilities and see if those can get you the boost you need. In our experience, shared caching is usually enough.
+
+However, there are instances where a remote daemon can make the most sense. Here are some examples:
+
+- You have a single, powerful build machine you would like to share with your development team
+- There is data closer to a remote machine than your development/CI environment, so you bring the build to the data
+- You are using Earthly in Kubernetes, and want to isolate the containers doing the actual building because they require privileged mode
+- You want to share a build machine (or cluster) with your CI environment and your developers
+- Your local computer does not have the capabilities to build the software (`docker` is missing, or you lack sufficient privileges, or it is simply not powerful enough)
```

Comment:
```suggestion
- Your local computer does not have the capabilities to build the software (`dockerd` is missing, or you lack sufficient privileges, or it is simply not powerful enough)
```

Reply:
Ehhh its really both. The CLI and the daemon. Ill clarify for both.

Reply:
Earthly requires the `docker` CLI anyway, right? e.g. for exporting tar images